addons/mail/static/src/new/composer/composer_model.js

```diff
@@ -35,4 +35,13 @@ export class Composer {
             _store: store,
         });
     }
+
+    reset() {
+        this.textInputContent = "";
+        this.selection = {
+            start: 0,
+            end: 0,
+            direction: "none",
+        };
+    }
```

Comment:
I want to avoid putting behaviour in models. Let's keep models pure data classes.
Doing so would keep models simple and also not let OOP find its way in discuss code, with all its subtle issues with owned functions (methods).